src/theory/arith/nl/nonlinear_extension.h

```diff
@@ -151,7 +146,7 @@ class NonlinearExtension
    * may have information regarding how to construct a model, in the case that
    * we determined the problem is satisfiable.
    */
-  bool modelBasedRefinement(std::vector<NlLemma>& mlems);
+  bool modelBasedRefinement();
```

Comment:
Can you update the documentation for this method (it doesnt add lemmas to lems)?